tests/test_elk_reporter.py

```diff
@@ -408,20 +408,12 @@ def test_2():
         """
     )
 
-    result = testdir.runpytest("--es-address=127.0.0.1:9200", "-v", "-s", "-n", "2")
+    result = testdir.runpytest("--es-address=127.0.0.1:9200", "-s", "-v", "-n", "2")
```

Comment:
it is different in order?

Reply:
not, I was editing this, try to make the requests_mock to work, but it can work across processes... which is a shame